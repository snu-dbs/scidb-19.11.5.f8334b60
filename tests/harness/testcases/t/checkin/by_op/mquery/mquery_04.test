--setup
--start-query-logging
create array s <v:int64>[i=1:*:0:1]
create array t <v:int64>[i=1:*:0:1]
create array u <v:int64>[i=1:*:0:1]

--test
mquery(insert(build(<v:int64>[i=1:9:0:1],1*i),s), \
               insert(s,t), \
               insert(t,u))

scan(s)
scan(t)
scan(u)

--cleanup
remove(s)
remove(t)
remove(u)
--stop-query-logging
