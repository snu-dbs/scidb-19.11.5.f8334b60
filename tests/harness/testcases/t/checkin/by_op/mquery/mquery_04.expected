SCIDB QUERY : <create array s <v:int64>[i=1:*:0:1]>
Query was executed successfully

SCIDB QUERY : <create array t <v:int64>[i=1:*:0:1]>
Query was executed successfully

SCIDB QUERY : <create array u <v:int64>[i=1:*:0:1]>
Query was executed successfully

SCIDB QUERY : <mquery(insert(build(<v:int64>[i=1:9:0:1],1*i),s), insert(s,t), insert(t,u))>
Query was executed successfully

SCIDB QUERY : <scan(s)>
{i} v
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5
{6} 6
{7} 7
{8} 8
{9} 9

SCIDB QUERY : <scan(t)>
{i} v
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5
{6} 6
{7} 7
{8} 8
{9} 9

SCIDB QUERY : <scan(u)>
{i} v
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5
{6} 6
{7} 7
{8} 8
{9} 9

SCIDB QUERY : <remove(s)>
Query was executed successfully

SCIDB QUERY : <remove(t)>
Query was executed successfully

SCIDB QUERY : <remove(u)>
Query was executed successfully

